src/coverage.ts

```diff
@@ -81,5 +104,5 @@ export const coverage = async ({ token, ghToken, summary }: Props) => {
     ...message
   })
 
-  return { status, rate, diff }
+  return { status, rate, diff, patchCoverage }
```

Comment:
We return a bunch of stuff here but we mainly need coverage, diff & patch.

Reply:
done